resources/keywords/deploy_instance_dev.resource

```diff
@@ -156,5 +156,11 @@ Text Message For A provider Account With No DB Instances Appears
 User Deploys MongoDB Instance Using CLI
     Deploy Db Instance Cli    Mongo    ${TEST TAGS}
 
+User Deploys CockroachDB Instance using CLI
+    Deploy Db Instance Cli    Cockroach    ${TEST TAGS}
+
+User Deploys CrunchyDB Instance using CLI
+    Deploy Db Instance Cli    crunchy    ${TEST TAGS}
+
```

Comment:
The idea is good, but the problem with this suggestion is that we are adding these keywords to the suits **according to isv type**, and what you are proposing will make us changing the structure of the files and re-engineer it because they will not be relevant anymore(tests/cockroachdb_import_connect_cli.robot, tests/crunchydb_import_connect_cli.robot).
WDYT?